common/src/main/res/values-fr/strings.xml

```diff
@@ -1445,31 +1445,31 @@ Code: {ERROR_CODE}"</string>
     <string name="verifier_check_mode_info_test_cert_text_3">"Ce mode ne permet pas de vérifier les «certificats light»."</string>
 
     <!-- wallet refresh button removal info box text -->
-    <string name="wallet_refresh_button_info_text_1">"Die Funktion zur Selbstprüfung von Zertifikaten («Refresh-Button») wurde entfernt."</string>
+    <string name="wallet_refresh_button_info_text_1">"Le bouton «rafraîchir» a été supprimée."</string>
 
     <!-- wallet refresh button removal info box text -->
-    <string name="wallet_refresh_button_info_text_2">"Sollte Ihr Covid-Zertifikat abgelaufen oder technisch ungültig sein, wird Ihnen dies direkt direkt auf dem Zertifikat angezeigt."</string>
+    <string name="wallet_refresh_button_info_text_2">"Si votre certificat COVID a expiré ou s'il n'est techniquement pas valable, cette information apparaît automatiquement sur le certificat."</string>
 
     <!-- wallet refresh button removal info box text -->
-    <string name="wallet_refresh_button_info_text_3">"Um Covid Zertifikate zu prüfen, muss der QR-Code mit einer Prüfapp gescannt werden."</string>
+    <string name="wallet_refresh_button_info_text_3">"La vérification des certificats COVID doit s'effectuer au moyen de l'application «COVID Certificate Check»."</string>
 
     <!-- wallet refresh button removal info box fat title above text 3 -->
-    <string name="wallet_refresh_button_info_fat_title_3">"Für Prüfende"</string>
+    <string name="wallet_refresh_button_info_fat_title_3">"Pour les personnes chargées du contrôle"</string>
 
     <!-- wallet refresh button removal info box link text -->
-    <string name="wallet_refresh_button_info_link_text">"Mehr erfahren"</string>
+    <string name="wallet_refresh_button_info_link_text">"Pour en savoir plus"</string>
 
     <!-- wallet refresh button removal info box link url -->
     <string name="wallet_refresh_button_info_link_url">"https://www.bag.admin.ch/bag/fr/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/covid-zertifikat/covid-zertifikat-pruefer-aussteller-technische-informationen.html#-1006172949"</string>
 
     <!-- text for info pop-up in wallet app describing 2G+ OK state -->
-    <string name="wallet_check_mode_info_2g_plus_ok_text">"Das Zertifikat ermöglicht den Zutritt zu Orten und Aktivitäten mit 2G+-Regelung. (Das Zertifikat für die Impfung oder Genesung ist weniger als 120 Tage alt.)"</string>
+    <string name="wallet_check_mode_info_2g_plus_ok_text">"L’accesso a strutture e manifestazioni è consentito alle persone per le quali sono trascorsi al massimo 120 giorni dal completamento del ciclo vaccinale, dalla somministrazione della dose di richiamo o dalla guarigione (sulla base di un test PCR). In questo caso non è necessario un ulteriore certificato di test."</string>
```

Comment:
This should be in french!

Reply:
Thanks, we already noticed this in https://github.com/admin-ch/CovidCertificate-App-Android/issues/378. These strings are also part of the app config, so we already fixed it server-side https://github.com/admin-ch/CovidCertificate-App-Config-Service/pull/45 (the string in the apps are fallbacks if the app has no network connection).

This branch is what we release as v3.2.0 last week, so we will merge it as is. We will fix the strings in a later PR.